src/components/MenuItem.js

```diff
@@ -38,11 +38,11 @@ const defaultProps = {
     shouldShowHeaderTitle: false,
     shouldParseTitle: false,
     wrapperStyle: [],
-    style: styles.popoverMenuItem,
+    style: undefined,
```

Comment:
This caused deploy blocker - https://github.com/Expensify/App/issues/32182
@rezkiy37 are you able to raise quick PR?

Reply:
@situchan, yes, I can fix it now.